package.json

```diff
@@ -26,10 +26,12 @@
     "ajv": "^6.12.4",
     "ant-design-vue": "^1.6.4",
     "csvtojson": "^2.0.10",
+    "follow-redirects": "^1.14.2",
     "less": "^3.12.2",
     "less-loader": "^6.2.0",
     "nuxt": "^2.14.3",
     "nuxt-ackee": "^2.0.0",
+    "replace-ext": "^2.0.0",
```

Comment:
not needed anymore, since javascript rewrite without imagemagick as a dependency is implemented